contracts/jobs/JobsManager.sol

```diff
@@ -198,8 +210,19 @@ contract JobsManager is ManagerProxyTarget, IVerifiable, IJobsManager {
         // Sender must be the job's broadcaster or elected transcoder
         require(job.broadcasterAddress == msg.sender || job.transcoderAddress == msg.sender);
 
+        // Decrement broadcaster's active jobs
+        broadcasters[msg.sender].activeJobs = broadcasters[msg.sender].activeJobs.sub(1);
+
+        uint256 endBlock = block.number.add(jobEndingPeriod);
+
         // Set end block for job
         job.endBlock = block.number.add(jobEndingPeriod);
```

Comment:
Should we use the variable `endBlock` in this assignment since we just calculated it one line above?